src/env.d/00-path

```diff
@@ -9,7 +9,10 @@
 
 [[ "$PATH" == *"${CREW_PREFIX}/sbin"* ]] || export PATH="${PATH}:${CREW_PREFIX}/sbin"
 
-[[ "$PATH" == *"${CREW_PREFIX}/share/musl/bin"* ]] || export PATH="${PATH}:${CREW_PREFIX}/share/musl/bin"
+# Only add musl bin to path if musl is installed.
+if [ -d "${CREW_PREFIX}/share/musl/bin" ]; then
+  [[ "$PATH" == *"${CREW_PREFIX}/share/musl/bin"* ]] || export PATH="${PATH}:${CREW_PREFIX}/share/musl/bin"
```

Comment:
`&&` instead of `||`?

Reply:
I believe the first expression checks if musl is already on the path, then if it fails the || runs the next expression, which adds it to the path

Reply:
What @Zopolis4 said. It works.

Reply:
Same code as used for the other path stuff.